bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Fan.java

```diff
@@ -45,21 +62,168 @@ static class ChannelConfiguration extends AbstractChannelConfiguration {
         protected @Nullable String commandTemplate;
         @SerializedName("command_topic")
         protected String commandTopic = "";
-        @SerializedName("payload_on")
-        protected String payloadOn = "ON";
```

Comment:
Double check: as `protected String payloadOff = "OFF";` is untouched, i noticed the `protected String payloadOn = "ON";` is removed as they are usually tied together / opposites. Was this intended?